scripts/index_d_ts

```diff
@@ -2470,6 +2472,37 @@ declare module '@sendbird/uikit-react/ui/ThumbnailMessageItemBody' {
   export default ThumbnailMessageItemBody;
 }
 
+declare module '@sendbird/uikit-react/ui/Toggle' {
+  export interface ToggleContextInterface {
+    checked?: boolean;
+    defaultChecked?: boolean;
+    disabled?: boolean;
+    reversed?: boolean;
+    onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
React.ChangeEventHandler<HTMLInputElement> doesnt work?

Reply:
Ah yes, that's good